src/utils/SceneUtils.ts

```diff
@@ -0,0 +1,49 @@
+import { BufferGeometry, Group, InstancedMesh, Material, Mesh, Object3D, Scene } from 'three'
```

Comment:
Could you separate the type imports from the regular imports?

Reply:
Sure